__init__.py

```diff
@@ -257,6 +257,14 @@ class Fast64_Properties(bpy.types.PropertyGroup):
 	oot: bpy.props.PointerProperty(type=OOT_Properties, name="OOT Properties")
 	settings: bpy.props.PointerProperty(type=Fast64Settings_Properties, name="Fast64 Settings")
 
+class Fast64_BoneProperties(bpy.types.PropertyGroup):
+	'''
+	Properties in bone.fast64.
```

Comment:
```suggestion
	Properties in bone.fast64. (bpy.types.Bone)
```
there's also PoseBone and EditBone so may as well make it clear

Reply:
done